From 4a1b4cc2fb0eddfa9951190f300a55d7edece77d Mon Sep 17 00:00:00 2001
From: ZhongWen <b47623@freescale.com>
Date: Mon, 9 Dec 2013 14:59:29 +0800
Subject: [PATCH 044/115] ENGR00288585 fix pop up error when exit WfdSink.apk

Add mTimer.cancel() method in onStop().
disable timer schedule when exit WfdSink.apk

Signed-off-by: ZhongWen <b47623@freescale.com>
---
 WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java b/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
index 9b9d9b3..52ab90d 100755
--- a/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
+++ b/WfdSink/src/com/freescale/wfdsink/WfdSinkActivity.java
@@ -443,6 +443,7 @@ public class WfdSinkActivity extends Activity implements SurfaceHolder.Callback,
         stopSearch();
         mWfdSink.dispose();
         mWfdSink = null;
+        mTimer.cancel();
 
     }
 
-- 
1.8.0

